### PR TITLE
Header22

### PR DESCRIPTION
**Pull request by BurcuAcan**

### CODE REVIEW

File: editor-components/header/header22/header22.module.scss [hunk 1]
```diff
@@ -1,147 +1,371 @@
...
-  min-height: calc(100vh - 80px);
+  height: 100%;
   background-color: var(--composer-html-background);
-  padding: 50px 0px;
   display: flex;
+  position: relative;
+  justify-content: center;
   align-items: center;
-  
+  padding: 20px 50px 20px 0;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:29]: can't you use gap value

File: editor-components/header/header22/header22.module.scss [hunk 2]
```diff
@@ -1,147 +1,371 @@
...
+            .middle-content,
+            .middle-content2,
+            .middle-content3 {
+              flex: 1;
+              width: 100%;    
+              position: relative;
+              left: calc(var(--composer-gap-md) * (-1));
+              // .mid-right-animation {
+              //   animation: rightToLeft 1s ease;
+              // }
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:108]: remove

File: editor-components/header/header22/header22.module.scss [hunk 3]
```diff
@@ -1,147 +1,371 @@
...
+              }
+            }
         
+            .middle-content3 {
+              left: var(--composer-gap-xl);
+              display: flex;
+              align-items: center;
+              justify-content: center;
+              .text-wrapper{
+                padding-left: 50px;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:165]: gap value?

File: editor-components/header/header22/header22.module.scss [hunk 4]
```diff
@@ -1,147 +1,371 @@
...
+        }
+
       }
     }
-    .middle {
-      flex: 1;
+    .nav-controls {
+      position: absolute;
+      right: 50px;
+      bottom: 20px;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:194]: gap value?

File: editor-components/header/header22/header22.module.scss [hunk 5]
```diff
@@ -1,147 +1,371 @@
...
-        .middle{
-          .content{
-            top: 25%;
-            width: 70%;
-            .title{
-              font-size: 3rem;
-              line-height: 30px;
-              margin-top: 8px;
-              margin-bottom: 8px;
+    padding: 20px 40px 20px 0;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:269]: gap value?

File: editor-components/header/header22/header22.module.scss [hunk 6]
```diff
@@ -1,147 +1,371 @@
...
+                    }
+                  }
+                }
+
+              }
+              .middle-content3, .middle-content2 {
+                left: var(--composer-gap-lg);
+                .text-wrapper{
+                  gap: var(--composer-gap-xs);
+                  padding-left: 40px;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:297]: gap value?

File: editor-components/header/header22/header22.module.scss [hunk 7]
```diff
@@ -1,147 +1,371 @@
...
             }
-            .button{
-              font-size: 1rem;
             }
-            .divider-underline {
-            margin-left: 4px;
+          }
+        }
+        .dots {
+          left: 40px;
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.module.scss:315]: gap value?

File: editor-components/header/header22/header22.tsx [hunk 8]
```diff
@@ -2,95 +2,390 @@ import * as React from "react";
...
+                              {item.button.map((buttonItem: any, indexButton: number) => {
+                                const buttonText = this.castToString(buttonItem.buttonText);
+                                if (buttonText) {
+                                  return (
+                                    <div className={this.decorateCSS("link-button-container")}>
+                                      <ComposerLink
+                                        className={this.decorateCSS("link-button")}
+                                        key={`hdr-22-${indexButton}`}
+                                        path={buttonItem.link}>
+                                        <button className={this.decorateCSS("button")}>{buttonText}</button>
```
Comment (Ayazelar) [inline editor-components/header/header22/header22.tsx:332]: use buttonItem.buttonText. otherwise the inline editor will not open

Comment (Ayazelar) [inline editor-components/header/header22/header22.tsx:332]: use base button

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

File: editor-components/header/header22/header22.module.scss (hunk 3 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:165]: for static container value

File: editor-components/header/header22/header22.module.scss (hunk 4 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:194]: these are also same resaon

File: editor-components/header/header22/header22.module.scss [hunk 9]
```diff
@@ -1,147 +1,371 @@
...
-            cursor: pointer;
+      }
+    }
+    .dots {
+      position: absolute;
+      top: 50%;
+      display: flex !important;
+      flex-direction: column;
+      padding: 0;
+      left: 50px;
```
[the remark this answers is not in the document]
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:215]: these are also same resaon

File: editor-components/header/header22/header22.module.scss (hunk 5 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:269]: these are also same reason

File: editor-components/header/header22/header22.module.scss (hunk 6 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:297]: these are also same resaon

File: editor-components/header/header22/header22.module.scss (hunk 7 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.module.scss:315]: these are also same resaon

File: editor-components/header/header22/header22.tsx (hunk 8 above)
Comment (BurcuAcan) [reply editor-components/header/header22/header22.tsx:332]: we are not using base botton for this button

**Pull request merged**